Gemfile

```diff
@@ -28,22 +28,14 @@ group :system_tests do
   gem 'beaker-puppet_install_helper', :require => false
 end
 
-is_x64 = Gem::Platform.local.cpu == 'x64'
-if is_x64
-  platform(:x64_mingw) do
-    gem "win32-dir", "~> 0.4.9", :require => false
-    gem "win32-process", "~> 0.7.4", :require => false
-    gem "win32-service", "~> 0.8.4", :require => false
-    gem "minitar", "~> 0.5.4", :require => false
-  end
-else
-  platform(:mingw) do
-    gem "win32-process", "~> 0.6.5", :require => false
-    gem "win32-service", "~> 0.7.2", :require => false
-    gem "minitar", "~> 0.5.4", :require => false
-    if RUBY_VERSION =~ /^1\./ then
-      gem "win32console", :require => false
-    end
+platforms :mswin, :mingw do
```

Comment:
> I'm wondering if there was another historical reason that the Gemfile looks like this that I'm forgetting

IIRC it looked that way because if you referenced `:x64_mingw` in the Gemfile, then older versions of bundler (1.3.5) running on 32-bit ruby would cause `bundle install` to explode, since it didn't recognize that platform identifier. We worked around this in a few different commits, but this one looks most relevant: https://github.com/puppetlabs/puppet/commit/b6a20930d589daadc36f7681fc40b1a0c6bb29c3.

That said it may not be required anymore, assuming we're using a newer version of bundler everywhere.